components/director/internal/tenantfetchersvc/handler.go

```diff
@@ -65,6 +89,43 @@ func NewTenantsHTTPHandler(subscriber TenantSubscriber, config HandlerConfig) *h
 	}
 }
 
+// NewTenantFetcherHTTPHandler returns a new HTTP handler, responsible for creation of on-demand tenants.
+func NewTenantFetcherHTTPHandler(fetcher TenantFetcher, config HandlerConfig) *handler {
+	return &handler{
+		fetcher: fetcher,
+		config:  config,
+	}
+}
+
+// FetchTenantOnDemand fetches CIS events for a provided subaccount and creates a tenant
+func (h *handler) FetchTenantOnDemand(writer http.ResponseWriter, request *http.Request) {
+	ctx := request.Context()
+
+	vars := mux.Vars(request)
+	tenantID, ok := vars[h.config.TenantPathParam]
+	if !ok {
+		log.C(ctx).Error("Tenant path parameter is missing from request")
```

Comment:
`log.C(ctx).WithError(err)...` <- this adds the error as `error` field in the json log message - it looks pretty in Kibana.
Same on L114